src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
   const [releaseDate, setReleasedate] = useState(
     editableMode ? formData?.release_date : ``
   );
+  return (
+    <Formik
+      initialValues={{
+        genres,
+        title,
+        overview,
+        posterPath,
+        runtime,
+        releaseDate,
+      }}
+      validationSchema={validationSchema}
+      onSubmit={(values) => {
+        onModalSubmit({
```

Comment:
```
onSubmit={(values) => {
    onModalSubmit({ ...values });
}}
```
`values` param will contain all actual form data

Reply:
except id and runtime should be a number)
but I can add the id field but it will hide